### PR TITLE
HW-3: Add verification 

### PR DESCRIPTION
**Pull request by rogerosha**
Hello! I fixed some bugs from the previous versions of code and added verification (email + password + checkbox) for  the page login:

image

it doesn't let me to go further without password (or email/checkbox):

image

after filling the fields it shows this page:

image

### CODE REVIEW

File: homeworks/ekaterina.sharova_rogerosha/PokemonProject/login.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                    <a href="#" class="navigation-item login">Login</a>
+                </li>
+            </ul>
+        </header>
+        <div class="content">
+            <h1 class="headline">Login page</h1>
+            <div class="authorization-container">
+                <section class="enter-content">
+                    <form action="./result.html">
+                        <div class="email-check">
```
Comment (anna-vrana-moc) [inline homeworks/ekaterina.sharova_rogerosha/PokemonProject/login.html:36]: It's an unnecessary wrap; you can style the label and input

File: homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  transition: .2s ease-in;
+}
+
+.real-checkbox:checked + .custom-checkbox::before {
+  transform: translate(-50%, -50%) scale(1);
+}
+
```
Comment (anna-vrana-moc) [inline homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css:181]: I don't see a checked state 
Screenshot 2023-10-28 at 11 38 05

File: homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+  text-decoration: none;
+  cursor: pointer;
+  display: flex;
+  width: 575px;
+  padding: 27px 258px;
+  justify-content: center;
+  align-items: center;
+  border-radius: 6px;
+  gap: 10px;
+}
```
Comment (anna-vrana-moc) [inline homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css:199]: Please add styles for the border

File: homeworks/ekaterina.sharova_rogerosha/PokemonProject/login.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                </li>
+                <li>
+                    <a href="#" class="navigation-item login">Login</a>
+                </li>
+            </ul>
+        </header>
+        <div class="content">
+            <h1 class="headline">Login page</h1>
+            <div class="authorization-container">
+                <section class="enter-content">
```
Comment (anna-vrana-moc) [inline homeworks/ekaterina.sharova_rogerosha/PokemonProject/login.html:34]: `<div class="authorization-container">` does nothing. Please remove it

File: homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css (hunk 2 above)
Comment (anna-vrana-moc) [reply homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css:181]: Please add a margin-bottom, like on the layout

**Review by anna-vrana-moc — CHANGES_REQUESTED** [review]

File: homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css (hunk 2 above)
Comment (rogerosha) [reply homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css:175]: > I don't see a checked state Screenshot 2023-10-28 at 11 38 05

I can't fix it :( 
Just don't know the way to make it work

Comment (anna-vrana-moc) [reply homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css:175]: You can either follow this [example](https://codepen.io/nikkz/pen/BzVBJo) or use the default checkbox. Both variants are accepted.

Comment (rogerosha) [reply homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css]: Now it works as it must work


image


And I also added extra task with filters for the star


image

**Review by anna-vrana-moc — APPROVED** [review]

**Pull request merged**